wiki/Tournaments/CHT/1/es.md

```diff
@@ -0,0 +1,476 @@
+- FreeMod
+  - [onoken - felys (Kloyd) \[Another\]](https://osu.ppy.sh/beatmapsets/235496#osu/546339)
+  - [MY FIRST STORY - Fukagyaku Replace (Broccoly) \[pishi's Extra\]](https://osu.ppy.sh/beatmapsets/383270#osu/870566)
+  - [Team "Hanayamata" - Hana wa Odore ya Irohaniho (Fycho) \[yf's Insane\]](https://osu.ppy.sh/beatmapsets/529815#osu/1128414)
+- Tiebreaker
+  - **[y0c1e - No buts! (Lazer) \[Adrenaline\]](https://osu.ppy.sh/beatmapsets/402278#osu/874939)**
+
+### Eliminatorias
+
+**[Descarga el paquete de mapas aquí! (66 MB)](https://www.mediafire.com/file/wn4sqp2gdkdfpw7/CHT_Mappool.rar/file)**
+
+- NoMod
+  - [Yonezawa Madoka - Responsibility Response (ShiraKai) \[Phantom\]](https://osu.ppy.sh/beatmapsets/276997#osu/627765)
+  - [xi - Akasha (Jemmmmy) \[U38's Another\]](https://osu.ppy.sh/beatmapsets/118768#osu/323883)
+  - [Memme - Pandemonium (sdafsf) \[Lasse's Extra: #8\]](https://osu.ppy.sh/beatmapsets/734190#osu/1550984)
+- Hidden
+  - [Lapix - LAST GAME (Shadren) \[Slayed's Insane\]](https://osu.ppy.sh/beatmapsets/524004#osu/1137713)
+  - [P\*Light - SAY BAY (MrMenda) \[Knight's Another\]](https://osu.ppy.sh/beatmapsets/807330#osu/1694422)
+- HardRock
+  - [Amatsuki - Outer Science (HelloSCV) \[Insane\]](https://osu.ppy.sh/beatmapsets/134767#osu/337873)
+  - [Guilty Kiss - KOWAREYASUKI (ktgster) \[Insane\]](https://osu.ppy.sh/beatmapsets/766867#osu/1619027)
+- DoubleTime
+  - [senya - Shunkan Everlasting (Naotoshi) \[Hard\]](https://osu.ppy.sh/beatmapsets/657870#osu/1414839)
+  - [Tia - Deal with the devil (-Mikan) \[ShadowX's Hard\]](https://osu.ppy.sh/beatmapsets/819239#osu/1882028)
+
+## Resultado de las partidas
+
+### Finales
+
+Sábado, 15 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **Hallo y caspi** | **7** | 1 | Ladrillo | [#1](https://osu.ppy.sh/community/matches/58471366) |
+
+Lunes, 17 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **United States** | **7** | 3 | Noveno Gaston | [#1](https://osu.ppy.sh/community/matches/58530443) |
+
+### Semifinales
+
+Viernes, 7 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| Hallo y Caspi | 3 | **6** | **United States** | [#1](https://osu.ppy.sh/community/matches/58274385) |
+
+Domingo, 9 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **Noveno Gaston** | **6** | 4 | Ladrillo | [#1](https://osu.ppy.sh/community/matches/58333712) |
+
+### Cuartos de Final
+
+Viernes, 31 Enero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **hallo y caspi** | **5** | 2 | Radical | [#1](https://osu.ppy.sh/community/matches/58108351) |
+
+Domingo, 2 Febrero 2020:
+
+| Equipo 1 |  |  | Equipo 2 | Link del match |
+| --: | :-: | :-: | :-- | :-- |
+| **Noveno Gaston** | **0** | -1 | Geometry Dash | *win by default* |
```

Comment:
"win by default" doesnt have translation ?

Reply:
It has but it doesn't really fit, they told me it's better to leave it as it is.